<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogaccounts</class>
 <widget class="QDialog" name="Dialogaccounts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>植此青绿小游戏</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>60</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labeltitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>336</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>130</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelbg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>336</width>
     <height>226</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>labelbg</zorder>
  <zorder>comboBox</zorder>
  <zorder>lineEdit</zorder>
  <zorder>pushButton</zorder>
  <zorder>labeltitle</zorder>
  <zorder>pushButton_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
